server/test/unit/server/async/test_scheduler.py

```diff
@@ -506,7 +507,7 @@ def test_debug_logging(self, mock__logger, mock_worker, mock_delete_worker):
         scheduler.CeleryProcessTimeoutMonitor().check_celery_processes()
         mock__logger.debug.assert_has_calls([
             mock.call('Checking if pulp_workers, pulp_celerybeat, or '
-                      'pulp_resource_manager processes are missing for more than 300 seconds'),
+                      'pulp_resource_manager processes are missing for more than 25 seconds'),
```

Comment:
Instead of hard-coding the timeout, can we make this PULP_PROCESS_TIMEOUT_INTERVAL